vscode-test failing with VS Code 1.89.1 (on Windows only)

At first I thought it was just an environment issue in Azure DevOps, but then I let my GitHub CI Workflow run and I'm getting this error _on Windows only_ (the Linux and macOS runs succeed):

```
> vscode-test

Downloading VS Code 1.89.1 from https://update.code.visualstudio.com/1.89.1/win32-x64-archive/stable
Downloading VS Code (138334713B)
Downloading VS Code [====--------------------------] 15%Downloading VS Code [============------------------] 41%Downloading VS Code [==================------------] 63%Downloading VS Code [==========================----] 87%Downloading VS Code [==============================] 100%Downloaded VS Code into D:\a\vscode-azurearcenabledmachines\vscode-azurearcenabledmachines\.vscode-test\vscode-win32-x64-archive-1.89.1
Downloaded VS Code into D:\a\vscode-azurearcenabledmachines\vscode-azurearcenabledmachines\.vscode-test\vscode-win32-x64-archive-1.89.1
Error: spawn EINVAL
    at ChildProcess.spawn (node:internal/child_process:414:11)
    at spawn (node:child_process:761:9)
    at file:///D:/a/vscode-azurearcenabledmachines/vscode-azurearcenabledmachines/node_modules/@vscode/test-cli/out/cli/platform/desktop.mjs:124:31
    at new Promise (<anonymous>)
    at PreparedDesktopRun.installDependentExtensions (file:///D:/a/vscode-azurearcenabledmachines/vscode-azurearcenabledmachines/node_modules/@vscode/test-cli/out/cli/platform/desktop.mjs:1[23](https://github.com/microsoft/vscode-azurearcenabledmachines/actions/runs/9198711227/job/25302002277?pr=66#step:6:24):16)
    at async PreparedDesktopRun.run (file:///D:/a/vscode-azurearcenabledmachines/vscode-azurearcenabledmachines/node_modules/@vscode/test-cli/out/cli/platform/desktop.mjs:70:9)
    at async runPreparedConfigs (file:///D:/a/vscode-azurearcenabledmachines/vscode-azurearcenabledmachines/node_modules/@vscode/test-cli/out/bin.mjs:117:31)
    at async main (file:///D:/a/vscode-azurearcenabledmachines/vscode-azurearcenabledmachines/node_modules/@vscode/test-cli/out/bin.mjs:33:20)
Error: Process completed with exit code 1.
```

https://github.com/microsoft/vscode-azurearcenabledmachines/actions/runs/9198711227/job/25302002277?pr=66

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -3,6 +3,7 @@
 ## TBA
 
 - Fix bailing multiple configs on first failure (fixes [#44](https://github.com/microsoft/vscode-test-cli/issues/44))
+- Fix extension install on Windows on modern Node versions (fixes [#43](https://github.com/microsoft/vscode-test-cli/issues/43))
 
 ## 0.0.9 - 2024-03-04
 

--- a/src/cli/platform/desktop.mts
+++ b/src/cli/platform/desktop.mts
@@ -170,7 +170,8 @@ class PreparedDesktopRun implements IPreparedRun {
 
     // todo@connor4312: have a nicer reporter here
     return new Promise<void>((resolve, reject) => {
-      const installer = spawn(cli, cliArgs, { stdio: 'pipe' });
+      const shell = process.platform === 'win32';
+      const installer = spawn(shell ? `"${cli}"` : cli, cliArgs, { stdio: 'pipe', shell });
       let output: string = '';
       installer.stdout.setEncoding('utf-8').on('data', (data) => {
         output += data;

